style.css

```diff
@@ -0,0 +1,73 @@
+body {
```

Comment:
Good job working on the styles but there are linters errors. A good way to solve this would be to set up linters in the project and run the linters error test to correct the errors.